syntax = "proto3";

service SquareService {
    rpc GetSquare (SquareRequest) returns (SquareResponse);
}

message SquareRequest {
    int32 number = 1;
}

message SquareResponse {
    int32 result = 1;
}